src/api/java/com/enderio/api/conduit/ticker/ConduitTicker.java

```diff
@@ -24,6 +24,10 @@ default int getTickRate() {
     // TODO: I'd argue this goes into ConduitType, and then you can use getTicker() if you need additional context from it.
     boolean canConnectTo(Level level, BlockPos conduitPos, Direction direction);
 
+    default boolean canHaveConnection(Level level, BlockPos conduitPos, Direction direction) {
```

Comment:
I don't think this method is adequately named - could you explain what the difference is from canConnectTo so we can figure a different name for this method? Also some javadoc may help :)

Reply:
Renamed it to `canForceConnect` with some javadoc. I don't really know what I was thinking with that name 😅

Reply:
That makes more sense, thanks!